ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetWorkTest.java

```diff
@@ -69,8 +69,7 @@ public void shouldReturnCorrectResultOnGenesisDAG() {
     final String[] expectedValue = {
       "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
       "0x0000000000000000000000000000000000000000000000000000000000000000",
-      "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
-      "0x0"
+      "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
```

Comment:
this is a meaningful change. The block number is no longer sent on eth_getWork ?

Reply:
this should be a separate PR to avoid mixing too many things and making sure folks are notified of the change in the release notes.

Reply:
Sure, I can send it separatedly.
I changed this when I realized that the _iPollo_ wasn't accepting the `eth_getWork` responses properly, then I checked the [spec](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getwork) I realized that it only specify three return elements. I would have expected that no parser would comply for having extra elements but it doesn't seems to be the case.
Do you think that being out of spec is enough for chaging this return value? otherwise I would need to implement a different protocol flavor.

Reply:
I guess it's fine to remove the fourth parameter. I am not completely sure where that came from, might be a misinterpretation of the spec for stratum1.

Reply:
I opened #5184 for this, but having the fourth value in the HTTP response was a requirement for a mining pool, so it cannot be removed. I reworked it to expose the specified three elements so it can be used in the stratum protocol but leave the HTTP response as is.